metrics/naming.go

```diff
@@ -1,10 +1,10 @@
 package metrics
 
-const serviceName = "pulse"
+const appName = "pulse"
 const requestsMetricBaseName = "requests"
 
 func metricName(name string) string {
-	return serviceName + "_" + name
+	return "bux_" + name
```

Comment:
maybe this prefixing is not needed?
maybe it should only be `http_requests*`
and if other metrics will came, then again we will decide on creation of it, how to name it?

Reply:
I wanted our (bux-toolkit) metrics to be quickly searchable by this prefix (in Grafana for example). 
For example here they added `api_` prefix here: https://prometheus.io/docs/practices/naming/#labels

Reply:
From the same link:
>...should have a (single-word) application prefix relevant to the domain the metric belongs to. The prefix is sometimes referred to as namespace by client libraries. For metrics specific to an application, the prefix is usually the application name itself. **Sometimes, however, metrics are more generic, like standardized metrics exported** by client libraries. Examples:
> - prometheus_notifications_total (specific to the Prometheus server)
> - process_cpu_seconds_total (exported by many client libraries)
> - **http_request_duration_seconds (for all HTTP requests)**

and this is why I'm talking about that, because often I saw `http_request_duration_seconds` as metric name in different systems I was participating in development, therefore it is for me a standard name for that metric.

Reply:
So, maybe a hybrid approach: 

- For common metrics in your bux-toolkit, adopt standard naming conventions (e.g., `http_request_duration_seconds`) to ensure ease of understanding 
- Use a custom `bux_` prefix for unique, system-specific metrics to clearly identify them and avoid name collisions. For example `bux_block_height_gauge`.

Reply:
Yes, this is something that I even thought at the beginning, I just focused on those http_requests now, because this was the first step.